examples/platform/find_users_with_grants.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# *******************************************************
+# Copyright (c) VMware, Inc. 2020-2021. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+# *******************************************************
+# *
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+"""Find users that have "interesting" grants (with profiles) and list them."""
+
+from cbc_sdk.helpers import build_cli_parser, get_cb_cloud_object
+from cbc_sdk.platform import User
+import sys
+
+
+def is_interesting(user):
```

Comment:
This should be `has_grant` as `is_interesting` is not descriptive

Reply:
Refactor -> Rename :)